modules/govuk_postgresql/templates/usr/local/bin/wal-e_env_sync

```diff
@@ -13,20 +13,23 @@ set -e
 # Redirect stdout and stderr to syslog
 exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
 
+# Disable Puppet so PostgreSQL isn't automatically restarted
+govuk_puppet --disable
```

Comment:
Hm, the `/var/lib/puppet` directory is owned by `puppet:puppet` and isn't world-readable so I'm not sure how the deploy or postgres user could test whether Puppet is disabled on the host. The file that gets created is `/var/lib/puppet/state/agent_disabled.lock`.